maps/away/ships/fed_humanitarian/fed_humanitarian_ship_ghostroles.dm

```diff
@@ -0,0 +1,87 @@
+/datum/ghostspawner/human/fed_humanitarian
+	short_name = "fed_humanitarian"
+	name = "Nralakk Humanitarian Worker"
+	desc = "You are a worker aboard a Nralakk Federation humanitarian vessel, sent to aid the Izweski Hegemony."
```

Comment:
every ghostspawner needs `welcome_text` with more information about what you are supposed to do after spawning there, any other non-obvious stuff about the ship or the role, or anything else
see scc scout or sol yacht for an example